conf/application.conf

```diff
@@ -145,8 +150,10 @@ microservice {
     yta.url = "http://localhost:9020/account"
     feedback.url = "http://localhost:9514/feedback/ATS/personal"
     login.url = "http://localhost:9025/gg/sign-in?continue=http://localhost:9217/annual-tax-summary"
-    login.paye.url = "http://localhost:9025/gg/sign-in?continue=http://localhost:9217/paye/annual-tax-summary"
     login-callback.url = "http://localhost:9217/annual-tax-summary"
+    paye.login.url = "http://localhost:9025/gg/sign-in"
```

Comment:
Do you need a continue bit?

Reply:
The continue url was appended twice so I've just removed it from this url 👍